notebooks/book1/03/gauss_infer_1d.ipynb

```diff
@@ -0,0 +1,336 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #29.        plt.xticks([-7, 0, 7])
</pre><p>Try commenting this line, may be then x axis will have more ticks and it would be easy to interpret values in x-axis.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyprobml-review/pyprobml-review/pull/21/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Done</p>